punishments.js

```diff
@@ -1081,6 +1088,13 @@ Punishments.monitorRoomPunishments = function (user) {
 			return `<<${room}>> (${punishDesc})`;
 		}).join(', ');
 
-		Monitor.log(`[PunishmentMonitor] ${user.name} currently has punishments in ${punishments.length} rooms: ${punishmentText}`);
+		if (numRoombans && (numRoombans + numBlacklists) >= 3) {
+			Punishments.lock(user, Date.now() + LOCK_DURATION, user.userid, `Autolocked for being banned from ${numRoombans + numBlacklists} rooms`);
```

Comment:
This could probably be more specific.

i.e., "Automatically locked for being banned from number public rooms."

And maybe even list the rooms in that, too?